modules/productcomments/views/templates/hook/post-comment-modal.tpl

```diff
@@ -0,0 +1,158 @@
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+
+<script type="text/javascript">
+  var productCommentPostErrorMessage = '{l s='Sorry, your review cannot be posted.' d='Modules.Productcomments.Shop' js=1}';
+</script>
+
+<div id="post-product-comment-modal" class="modal fade product-comment-modal" role="dialog" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <p class="h4 mb-0">{l s='Write your review' d='Modules.Productcomments.Shop'}</p>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <form id="post-product-comment-form" action="{$post_comment_url nofilter}" method="POST">
+          <div class="row">
+            <div class="col-md-2 col-sm-2">
+              {if isset($product) && $product}
+                {block name='product_flags'}
+                  <ul class="product-flags">
+                    {foreach from=$product.flags item=flag}
+                      <li class="product-flag {$flag.type}">{$flag.label}</li>
+                    {/foreach}
+                  </ul>
+                {/block}
+
+                {block name='product_cover'}
+                  <div class="product-cover">
+                    {if $product.cover}
+                      <img class="js-qv-product-cover rounded" src="{$product.cover.bySize.medium_default.url}" alt="{$product.cover.legend}" title="{$product.cover.legend}" style="width:100%;" itemprop="image">
+                    {else}
+                      <img src="{$urls.no_picture_image.bySize.large_default.url nofilter}" style="width:100%;" class="rounded">
+                    {/if}
+                  </div>
+                {/block}
+              {/if}
+            </div>
+            <div class="col-md-4 col-sm-4">
+              <h3>{$product.name}</h3>
+              {block name='product_description_short'}
+                <div itemprop="description">{$product.description_short nofilter}</div>
+              {/block}
+            </div>
+            <div class="col-md-6 col-sm-6">
+              {if $criterions|@count > 0}
+                <ul id="criterions_list">
+                  {foreach from=$criterions item='criterion'}
```

Comment:
We can modify the module so it uses normal radio inputs behind the stars like here - https://github.com/prestarocket-agence/classic-rocket/blob/2.1.3/modules/productcomments/views/templates/hook/post-comment-modal.tpl

Less javascript needed ;-)

Reply:
IMO star style is better.

Reply:
![Screenshot 2022-04-27 at 12-07-45 Hummingbird printed sweater](https://user-images.githubusercontent.com/85633460/165563236-4fc91e9d-d6af-463e-b019-4a9107e9b7d7.png)

Reply:
@mparvazi It would still use stars, but it would not require the jquery plugin to do it ;-)

Reply:
Let's keep it like that for the moment, should be done in another PR